### DISCUSSION TITLE
Where can I learn more about the CRDs available in Argo?

### DISCUSSION THREAD

**Discussion by ramanNarasimhan77** [argoproj/argo-workflows#11166] [Q&A] [answered]
I am familiar with 
```
clusterworkflowtemplates.argoproj.io
cronworkflows.argoproj.io
workflows.argoproj.io
workflowtemplates.argoproj.io
```

But I would like to understand more about the below CRDs which are more recent additions. Looking through the documentation, I could not find sections explaining these CRDs or get pointers on how it gets used.

```
workfloweventbindings.argoproj.io
workflowtaskresults.argoproj.io
workflowtasksets.argoproj.io
workflowartifactgctasks.argoproj.io
```

**Comment by terrytangyuan** [comment, marked as answer]
Duplicate https://github.com/argoproj/argo-workflows/issues/11167
